pkg/cluster/cluster.go

```diff
@@ -120,6 +124,15 @@ func New(ctx context.Context, log *logrus.Entry, _env env.Interface, db database
 
 	storage := storage.NewManager(_env, r.SubscriptionID, fpAuthorizer)
 
+	// TODO: always set hive once we have it everywhere in prod and dev
+	var hr hive.ClusterManager
```

Comment:
I don't super love the idea of landing this -- but I guess it's harmless for now.